code/datums/abilities/critter/nascent.dm

```diff
@@ -0,0 +1,89 @@
+/datum/targetable/critter/nascent/become_commander
+	name = "Become commander"
+	desc = "Become a hallberd-wielding skeleton and summon more bone rattlers."
+	icon_state = "choose_skeleton"
+	cooldown = 0
+	targeted = 0
+	var/border_icon = 'icons/mob/wraith_ui.dmi'
+	var/border_state = "harbinger_frame"
+
+	cast()
+		if (..())
+			return 1
+		var/mob/wraith/W = null
+		if(istype(holder.owner, /mob/living/critter/wraith/nascent))
+			var/mob/living/critter/wraith/nascent/N = holder.owner
```

Comment:
single character variable names are to be avoided, especially when you re-define them later D: